docs/troubleshooting.md

```diff
@@ -108,19 +116,23 @@ If everything seems healthy, and still no packages show up in the UI, it could b
+If quay.io is unavailable, the `MESSAGE` will usually read `unknown error (status 404)`. If the steps listed in this section does not help debug the problem, please reach out to one of the forums listed in [Where to go for help](#where-to-go-for-help).   
 
-Note: A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [end point](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the end point is specified as `https://quay.io/cnr`.
+## Operator(s) in an OperatorSource fail to show up in the UI
 
-## OperatorSource `MESSAGE` reads unknown error (status 404)
+If operators in a particular OperatorSource fail to show up in the UI, it could be because those operators had parsing errors and were ignored by the registry pod. To inspect the corresponding registry pod logs, first identify the name of the registry pod for the OperatorSource with `oc get pods -n openshift-marketplace` (the name of the pod should be of the format `<operator-source-name>-<random-characters>`). Get the logs for the pod with `oc logs <identified-pod-name> -n openshift-marketplace`.
 
-Some of the common reasons we have encountered that caused this `MESSAGE` are quay.io being unavailable or the wrong end point being listed in the OperatorSource. If the steps listed [here](#operatorSource-failing-to-download-from-datastore) does not help debug the problem, please reach out to one of the forums listed in [Where to go for help](#where-to-go-for-help).   
+If there are private app-registry repositories in your namespace, not specifying the authenticationToken in the OperatorSource CR will result in them not being listed. Please follow the steps [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md) for adding the token to the CR.
 
 ## CatalogSourceConfig stuck in Configuring phase
 
-If you are trying to [install an operator from the cli](https://github.com/operator-framework/operator-marketplace#installing-an-operator-using-marketplace), the packages listed in the CatalogSourceConfig must have valid package names. If you have private app-resgistries, please follow the steps listed [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md).
+If you are trying to [install an operator from the cli](https://github.com/operator-framework/operator-marketplace#installing-an-operator-using-marketplace), the packages listed in the CatalogSourceConfig must have valid package names. For guidelines on what contitutes as a valid package name, please see [this section](#conflicting-package-names). Note that the `packageName` of your application should be unique, i.e it cannot have the same `packageName` as an existing application in any other OperatorSource in the cluster.  
```

Comment:
@aravindhp @awgreene @kevinrizza @SamiSousa I'm trying to think about reasons I have seen that has caused the csc to get stuck in the configuring phase but I'm coming up blank. Any input will be helpful here.

Reply:
You can leave it out for now then.